api/v1/controllers/lenses.js

```diff
@@ -39,7 +39,9 @@ function updateLensDetails(seqObj) {
     if (seqObj.hasOwnProperty('sourceName')) {
       seqObj.name = seqObj.sourceName;
     } else {
-      throw new apiErrors.ValidationError();
+      throw new apiErrors.ValidationError({
+        explanation: 'name or source name is required in lens json.',
```

Comment:
I think name is always required in lens.json, but sourceName is not a valid option for lens.json. Users should never be posting sourceName--that's for us internally, i.e. if a user overrides the name when posting the lens, then we store the name from lens.json in the "sourceName" column in the db.